src/current/_includes/releases/v23.1/v23.1.3.md

```diff
@@ -0,0 +1,147 @@
+## v23.1.3
+
+Release Date: June 13, 2023
+
+{{site.data.alerts.callout_danger}}
+A change included in this patch release was discovered to [contain a bug](https://github.com/cockroachdb/cockroach/issues/104798) that can affect clusters upgrading from [v22.2.x](v22.2.html). This can result in any jobs that were running during the upgrade to hang or fail to run after the upgrade is finalized. Users upgrading from v22.2.x are advised to use [v23.1.2](#v23-1-2) to upgrade, or to set the [`cluster.preserve_downgrade_option`](../v23.1/upgrade-cockroach-version.html#step-3-decide-how-the-upgrade-will-be-finalized) cluster setting to prevent finalization of the upgrade until they can do so using v23.1.4.
```

Comment:
```suggestion
A [bug](https://github.com/cockroachdb/cockroach/issues/104798) was discovered in a change included in this patch  that can affect clusters upgrading from [v22.2.x](v22.2.html). In an affected cluster, jobs that were running during the upgrade could hang or fail to run after the upgrade is finalized. Users upgrading from v22.2.x are advised to use [v23.1.2](#v23-1-2) to upgrade, or to set the [`cluster.preserve_downgrade_option`](../v23.1/upgrade-cockroach-version.html#step-3-decide-how-the-upgrade-will-be-finalized) cluster setting to delay finalization of the upgrade until they can upgrade to v23.1.4.
```
WDYT? This might be a little more direct and simpler to read.